lib/codec/plain.ts

```diff
@@ -146,17 +146,25 @@ function encodeValues_INT96(values: number[]) {
   return buf;
 }
 
-function decodeValues_INT96(cursor: Cursor, count: number) {
+function decodeValues_INT96(cursor: Cursor, count: number, opts?: Options) {
   const values = [];
+  // Default to false for backward compatibility
+  const treatAsTimestamp = opts?.treatInt96AsTimestamp === true;
 
   for (let i = 0; i < count; ++i) {
-    const low = INT53.readInt64LE(cursor.buffer, cursor.offset);
-    const high = cursor.buffer.readUInt32LE(cursor.offset + 8);
+    const nanosSinceMidnight = INT53.readInt64LE(cursor.buffer, cursor.offset);
```

Comment:
These should be renamed back to `low` and `high` since it is not always going to be a timestamp value.

Reply:
good call